server.js

```diff
@@ -13,6 +13,7 @@ const login = require("./app/controllers/login");
 const logout = require("./app/controllers/logout");
 const renderLogin = require("./app/controllers/renderLogin");
 const serveNotFound = require("./app/controllers/serveNotFound");
+const matches = require("./app/controllers/match");
```

Comment:
Looks like you `.js` file doesn't match the name you give to your `const`... Again, I would rename it to something like "renderLikedPersons" so it describes it's purpose. Also you want to be consistent in your naming, so don't go naming it match as your filename and matches when importing it somewhere else. I also saw that your function is called "showUsers", it's pretty inconsistent this way, I would name everything the same everywhere. So if you write a module (function) called "showUsers" then also name the `.js` file like "showUsers.js" and import it as `const showUsers = require("./app/controllers/showUsers")`